src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -6,64 +6,196 @@
 #
 import random as _random
 import turicreate as _tc
+import turicreate.toolkits._internal_utils as _tkutl
 from turicreate import extensions as _extensions
 from turicreate.toolkits._model import CustomModel as _CustomModel
+from turicreate.toolkits._model import PythonProxy as _PythonProxy
+from turicreate.toolkits.object_detector.object_detector import ObjectDetector as _ObjectDetector
 
 def create(dataset,
            target,
+           augmentation_mode="auto",
+           flip_horizontal=True,
+           flip_vertical=True,
            backgrounds=None,
-           feature=None,
            batch_size=0,
            max_iterations=0,
            seed=None,
-           verbose=True):
-    model = _extensions.one_shot_object_detector()
+           verbose=True,
+           **kwargs):
+    """
+    Create a :class:`ObjectDetector` model.
+
+    Parameters
+    ----------
+
+    dataset : SFrame | tc.Image
```

Comment:
If an SFrame is provided, how does the toolkit know which column contains the image to use?